### DISCUSSION TITLE
Compatibility with iTop 3.0.0 beta

### DISCUSSION THREAD

**Discussion by Schirrms** [Molkobain/itop-datacenter-view#13] [General]
Hi Molkobain,

I suspect that you tried your wonderful extention with iTop 3.0.0 :) 

Did you encounter any troubles ? 

Thanks,

Schirrms.

**Comment by Molkobain** [comment]
Hello Schirrms,

I tested it briefly but still waiting for a more stable release before migrating all the extensions. So far for this one here are the known issues:
- A PHP deprecation notice will be visible on the graphical view if you enabled the E_DEPRECATED in your php.ini
- A JS file deprecation (qTip lib) which blocks elements to be rendered in a Rack/Enclosure
- Graphical view background color is grey instead of white
- Some Font Awesome icons might not be the excepted one as the extension embeds a older version than the one embedded in iTop

All those will require a new extension version to correctly fix them, should be available around the same time as the iTop 3.0 release :)

Feel free to send your feedback if you see anything else!

Cheers,
Guillaume

**Comment by Molkobain** [comment]
Edited message above to add JS file deprecation found par @Schirrms on [SF](https://sourceforge.net/p/itop/discussion/third-party-extensions/thread/8f13acaff2/?limit=25)

**Comment by Molkobain** [comment]
Update: Migration of the extension has started!

✔ A PHP deprecation notice will be visible on the graphical view if you enabled the E_DEPRECATED in your php.ini
✔ A JS file deprecation (qTip lib) which blocks elements to be rendered in a Rack/Enclosure
✔ Graphical view background color is grey instead of white

Will keep you posted when a beta version is available (within the week)

**Comment by Molkobain** [comment]
Hello Schirrms,

You can now download v1.9.0 from the releases section, which should be perfectly compatible with iTop 2.7 and 3.0+ 🚀

Let me know,
Guillaume
